src/components/NewsPanel.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import LoginPanel from './LoginPanel.jsx'
+
+const NewsPanel = () => (
+  <div class="jumbotron banner center-block">
+    <div class="panel panel-default">
+      <div class="panel-body">
+        <div class="row">
+          <div class="col-md-8">
+            <h1> Latest News </h1>
+            <ul>
```

Comment:
We could convert these `<li>` into a React Component like `NewsItem` or something. This is fine for now.

Reply:
#88